data.json

```
@@ -0,0 +1,899 @@
+{
+  "assessedAt": "2025-04-22T11:00:00Z",
+  "assessedBy": {
+    "id": "mailto:mlieberman85@gmail.com",
+    "comment": "Automated scan + manual review based on internal checklist #789"
+  },
+  "assessmentComment": "Full Self-Assessment for OpenSSF Security Baseline for WTF Frontend",
+  "controls": [
+    {
+      "id": "OSPS-AC-01.01",
+      "conclusion": "non-compliant",
+      "evidence": [
+        {
+          "humanEvidence": {
+            "name": "manual-inspection",
+            "humanDescription": "Examined AuthN MFA requirements for the GitHub Organization and found lack of enforcement. Changing the setting could be disruptive to the organization and will require further planning to mitigate."
+          }
+        }
```

The [schema I proposed in BaselinePredicate](https://docs.google.com/document/d/16zwe3eNwExvnaXLDUrkGNineZzB0r8arKE3G-cvNU0E/edit?tab=t.0#heading=h.789n7qdnlkan) would render this as:

```suggestion
        {
          "humanDescription": "Examined AuthN MFA requirements for the GitHub Organization and found lack of enforcement. Changing the setting could be disruptive to the organization and will require further planning to mitigate."
        }
```

---

Yep. I modeled it this way, so on the code generation side we could rely on concrete types, vs sum types